django/core/management/templates.py

```diff
@@ -76,12 +75,8 @@ def handle(self, app_or_project, name, target=None, **options):
             top_dir = path.join(os.getcwd(), name)
             try:
                 os.makedirs(top_dir)
-            except OSError as e:
-                if e.errno == errno.EEXIST:
-                    message = "'%s' already exists" % top_dir
-                else:
-                    message = e
-                raise CommandError(message)
+            except FileExistsError:
+                raise CommandError("'%s' already exists" % top_dir)
```

Comment:
Unless I missed something:

- before: all `OSError` exceptions are converted to `CommandError`; in addition a specific message is added when the file already exists
- after: only `FileExistsError` is converted to `CommandError`